S/SZ/build_tarballs.jl

```diff
@@ -80,7 +80,8 @@ dependencies = [
                platforms=filter(!Sys.isbsd, platforms)),
     Dependency(PackageSpec(name="LLVMOpenMP_jll", uuid="1d63c593-3942-5779-bab2-d838dc0a180e");
                platforms=filter(Sys.isbsd, platforms)),
-    Dependency("HDF5_jll"; platforms=hdf5_platforms),
+    # Updating to a newer HDF5 version is likely possible without problems but requires rebuilding this package
```

Comment:
This is already on `master` https://github.com/JuliaPackaging/Yggdrasil/commit/d995ccc02f0de759380aea9d005ca9cf51a146b6 or am I missing something?

Reply:
I think this part is already merged from #6707 . I did this before the automerge completed.